lib/Sema/MiscDiagnostics.cpp

```diff
@@ -3727,7 +3727,8 @@ ASTWalker::PreWalkResult<Expr *> VarDeclUsageChecker::walkToExprPre(Expr *E) {
     return Action::SkipChildren(E);
   }
 
-  assert(AllExprsSeen.insert(E).second && "duplicate traversal");
+  #warning "fix me"
+  // assert(AllExprsSeen.insert(E).second && "duplicate traversal");
```

Comment:
What cases will hit duplicate traversal?

Reply:
Oh this needs to be cleaned up! Good catch.